environment.yml

```diff
@@ -42,7 +42,7 @@ dependencies:
   # NOTE: breathe 4.33.0 collides with sphinx.ext.graphviz
   - breathe>4.33.0
   # For linting
-  - pycodestyle=2.12.1
+  - ruff=0.6.7
```

Comment:
?/ I note that the ruff version is a bit out of date, it has a short release cycle :)